Срёт в логи сервера клиенто-парашей

[18:49:47][ERROR] Client is lacking preferences: AmiClerick (/client)
и таких по 200, самых разных. Никак не мешает кроме того что занимает чуть ли не половину с четвертью логов ВООБЩЕ.

_Этот иссуй был создан автоматически по [сообщению из дискорда](https://discordapp.com/channels/414832443384659968/606411392811139072/763103610232897567). Автор: **JamsMor#1622**._

Сервер упал с 200к таких логов за 15-20 минут:

```
Mar 21 23:53:13 onyxhost DreamDaemon[1084009]: [23:53:13][ERROR] Client is lacking preferences: Novan13 (/client) ([0x5000008])
Mar 21 23:53:13 onyxhost DreamDaemon[1084009]: [23:53:13][ERROR] Client is lacking preferences: Novan13 (/client) ([0x5000008])
Mar 21 23:53:13 onyxhost DreamDaemon[1084009]: [23:53:13][ERROR] Client is lacking preferences: Novan13 (/client) ([0x5000008])
Mar 21 23:53:13 onyxhost DreamDaemon[1084009]: [23:53:13][ERROR] Client is lacking preferences: Novan13 (/client) ([0x5000008])
```

Повышаю приоритет, слишком уж серьезный баг.

Скорее всего затрагивает #2501

--- a/code/modules/client/preference_setup/global/05_settings.dm
+++ b/code/modules/client/preference_setup/global/05_settings.dm
@@ -99,14 +99,22 @@
 	return ..()
 
 /client/proc/get_preference_value(preference)
-	if(prefs)
-		var/datum/client_preference/cp = get_client_preference(preference)
-		if(cp)
-			return prefs.preference_values[cp.key]
-		else
-			return null
+	if (!prefs)
+		log_error("[ckey]'s preferences did not load. Trying to fix.")
+		prefs = SScharacter_setup.preferences_datums[ckey]
+
+	if (!prefs)
+		log_error("[ckey]'s preferences are broken. Creating new one.")
+		prefs = new /datum/preferences(src)
+
+	if (!prefs)
+		CRASH("Can't create preferences for [ckey].")
+
+	var/datum/client_preference/cp = get_client_preference(preference)
+	if(cp)
+		return prefs.preference_values[cp.key]
 	else
-		log_error("Client is lacking preferences: [log_info_line(src)]")
+		return null
 
 /client/proc/set_preference(preference, set_preference)
 	var/datum/client_preference/cp = get_client_preference(preference)

